packages/storage-plus/src/de.rs

```diff
@@ -1,13 +1,9 @@
-// TODO: Remove along with IntKey
-#![allow(deprecated)]
-
 use std::array::TryFromSliceError;
 use std::convert::TryInto;
 
 use cosmwasm_std::{Addr, StdError, StdResult};
 
-use crate::int_key::CwIntKey;
-use crate::keys::{IntKey, TimestampKey};
+use crate::int_key::IntKey;
```

Comment:
I'm a little worried about this renaming introducing confusion. The module name is different, though. Perhaps that's enough to disambiguate / clarify this change.

Reply:
Had similar doubts.
All right, I reverted that commit - now it could be cherry-picked some time in the future.

Reply:
Now tracked in #775.